### ISSUE TITLE
Exception: Unable to find attributes in C:\Users\\AppData\Local\Temp\terraform_src\website\docs\r\vpc.html.markdown

### ISSUE THREAD

**Issue by ptthuc26** [DontShaveTheYak/cf2tf#202]
please help me check this issue
![image](https://github.com/DontShaveTheYak/cf2tf/assets/55550757/9c9381b1-34b0-44b0-9492-0bb5a1bb9b5c)

**Comment by Sab0tag3d** [comment]
The error is in `terraform\doc_file.py`

```
        try:
            attributes = parse_section("Attributes Reference", file)
        except Exception as e:
            raise Exception(f"Unable to find attributes in {file.name}") from e
```

It should be "Attribute" instead of "Attributes"

You can easily fix it by changing in library file (in your case it is C:\bla-bla-bla\terraform\doc_file.py string 19)

**Comment by shadycuz** [comment]
@Sab0tag3d Not it the latest version of the docs https://registry.terraform.io/providers/hashicorp/aws/3.3.0/docs/resources/vpc

The issue is that: https://github.com/DontShaveTheYak/cf2tf/blob/c46872d6952f067211adce73f6fc001a87ad713c/src/cf2tf/terraform/doc_file.py#L116-L118

`raise Exception()` was for testing 🙃 and should not have been added.

**Comment by Sab0tag3d** [comment]
Maybe my explanation was insufficient, I will try again :)

I had installed latest version of tool and it downloaded terraform documentation to path:
`/tmp/terraform_src/website/docs/r/...`

Then when I'd ran it, in debag mod I saw:
```
debug: Unable to find section Attributes Reference in tmp/terraform_src/website/docs/r/cloudwatch_event_rule.html.markdown
```

If will open this file, we would see:
```
## Attribute Reference

This resource exports the following attributes in addition to the arguments above:

* `id` - The name of the rule.
* `arn` - The Amazon Resource Name (ARN) of the rule.
* `tags_all` - A map of tags assigned to the resource, including those inherited from the provider [`default_tags` configuration block](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#default_tags-configuration-block).
```

As you can see, paragraph name is `## Attribute Reference`, howeverб tool is searching for Attribute**s**:
https://github.com/DontShaveTheYak/cf2tf/blob/c46872d6952f067211adce73f6fc001a87ad713c/src/cf2tf/terraform/doc_file.py#L19C12-L19C69

When I'd changed string to `"Attributes Reference"` it resolved the issue.

**Comment by ptthuc26** [comment]
I can not doc_file.py file. i install this tool through pip package

**Comment by ptthuc26** [comment]
@Sab0tag3d Can you instruct me for how to edit doc_file.py file?

**Comment by shadycuz** [comment]
@Sab0tag3d You are correct

@ptthuc26 Terraform decided to rename Attributes to Attribute in https://github.com/hashicorp/terraform-provider-aws/pull/32555 which broke our ability to parse correctly. This has been fixed in [v0.6.1](https://github.com/DontShaveTheYak/cf2tf/releases/tag/0.6.1).
